src/components/navbar/navbar.component.tsx

```diff
@@ -1,6 +1,8 @@
 import React from "react";
 import UserAvatarFilledAlt20 from "@carbon/icons-react/es/user--avatar--filled--alt/20";
+import { Switcher20 } from "@carbon/icons-react";
```

Comment:
See above - import from respective module (see `UserAvatarFilledAlt20 `)

Reply:
alright, I'll do that now